plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -108,31 +109,44 @@ type resourceRequest struct {
 func getResourceRequest(pod *api.Pod) resourceRequest {
 	result := resourceRequest{}
 	for ix := range pod.Spec.Containers {
-		limits := pod.Spec.Containers[ix].Resources.Limits
-		result.memory += limits.Memory().Value()
-		result.milliCPU += limits.Cpu().MilliValue()
+		requests := pod.Spec.Containers[ix].Resources.Requests
```

Comment:
So right now LimitRange is applying Limits, and scheduler was working against that.  Who is setting requests here?  I would have expected a change to have something applied as part of this PR.